lang/LangPrimSource/SC_Speech.mm

```diff
@@ -108,8 +108,9 @@ int prInitSpeech(struct VMGlobals *g, int numArgsPushed){
 	OSErr theErr = noErr;
 	//PyrSlot *a = g->sp-1;
 	PyrSlot *b = g->sp;
-    int chan;
-    slotIntVal(b, &chan);
+    int chan, err;
+    err = slotIntVal(b, &chan);
```

Comment:
btw, it is always a good idea to declare the variables exactly where they are used. having to declare all local variables at the beginning of the function is a pre-c99 requirement for c, but sc is using c++.
especially, it doesn't make the code exactly readable